src/tool_operate.c

```diff
@@ -1229,6 +1229,17 @@ static CURLcode single_transfer(struct GlobalConfig *global,
 
         use_proto = url_proto(per->this_url);
 
+#ifndef EXIT_MAY_BE_LEAKY
```

Comment:
Isn't this name the other way around? If this is defined, then it will not leak. But perhaps more importantly: since nothing sets this define maybe we should just not use it and let someone who actually needs it add it later?

Reply:
You're obviously right, sorry. Removed the ifdef as suggested.